automated install/basic-install.sh

```diff
@@ -2160,7 +1992,9 @@ Your Admin Webpage login password is ${pwstring}"
 IPv4:	${IPV4_ADDRESS%/*}
 IPv6:	${IPV6_ADDRESS:-"Not Configured"}
 
-If you set a new IP address, you should restart the Pi.
+If you have not done so already, the above IP should be set to static. Depending on your operating system, there are many ways to do this.
```

Comment:
We should print this at least in bold-face. Maybe even in red.

Reply:
Remember that it doubles already with the whiptail prompt at install start: https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L654
But that is not necessarily a bad thing.

Reply:
Require positive acknowledgement of the conditions. ("Type `P` to confirm ")

Reply:
Interactive action, more than just hitting "return" is probably the best if you worry about user locking out themselves from SSH access after reboot.

Instead of having two places where this info is prompted + interactive input, what about changing the msgbox from welcome dialogues into a yesno with "no" as default, so that an interactive switch to "yes" is required. The information/text that is added here could be moved into the whiptail as replacement, and then as question:
> Do you acknowledge this information and configure your system accordingly?

If "no" is chosen (respectively return pressed), the installer could exit and print again the information to apply a static or reserved IP (if router is DHCP server) before installing Pi-hole.

For unattended installs the new `printf` below could stay and there red colour and bold can be done as well.

Reply:
It would need to be in a whiptail, as we cannot wait for user input when piping the curled script to bash. (Unless we changed the install command to `bash -c "$(curl -sSL http://install.pi-hole.net)"` - but that's a bit more of a mouthful)

Reply:
I misread your comment.

I kind of like the idea of the first whiptail requiring a switch to `yes` from a defaulted `no`